action.yaml

```diff
@@ -9,10 +9,20 @@ inputs:
     description: The command to run
     required: true
   debug:
-    description: Whether to run mpyl in debug mode
+    description: Whether to run MPyL in debug mode
+    required: false
+  environment:
+    description: The environment this action runs in
+    required: true
+  version:
+    description: The version to be released with this workflow run
```

Comment:
MPyL itself does not inherently release the version, so what would you say this variable does? Perhaps something like 

`The version this Action runs with`?

Reply:
Yeah it's hard to describe, this is merely used to pass the `TAG` env variable to mpyl (and then mpyl uses it for a bunch of different stuff). I couldn't come up with a good description so I just copy/pasted it from one of the other workflows, for consistency 🤷